Code/Framework/AzFramework/AzFramework/Windowing/NativeWindow.cpp

```diff
@@ -284,6 +307,61 @@ namespace AzFramework
         return false;
     }
 
+    void NativeWindow::Implementation::SetEnableCustomizedResolution(bool enable)
+    {
+        if (m_enableCustomizedResolution != enable)
+        {
+            // If disable customized resolution, we need to update render resolution based on client area size
+            WindowSize resolution;
+            if (!enable)
+            {
+                SetRenderResolution(GetClientAreaSize());
+            }
+            m_enableCustomizedResolution = enable;
+            if (enable)
```

Comment:
Tiny nit: If you move the `m_enableCustomizedResolution = enable;` down to the bottom, you could make the `if (enable)` just an `else`.

If it needs to appear exactly where it is due to side effects of it being set before SetRenderResolution() or OnResolutionChanged, then please add a comment describing that.

Reply:
I will add comment